adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -1174,7 +1176,40 @@ func CreateTokenRoute() *routev3.Route {
 
 // CreateJwksEndpoint generates a route for JWKS /.wellknown/jwks endpoint
 func CreateJwksEndpoint() *routev3.Route {
-	return createStaticRoute(jwksPath, "/jwks", extAuthzHTTPClusterName)
+	conf, _ := config.ReadConfigs()
+	route := createStaticRoute(jwksPath, "/jwks", extAuthzHTTPClusterName)
```

Comment:
shall we introduce constant for "jwks" path

Reply:
+1